samples/StaticHtmlComposites/index.html

```diff
@@ -47,7 +47,11 @@
         token: token
       },
       document.getElementById('chat-container'),
-      { visualElements: { showParticipantPane: true, showTopic: false } }
+      {
+        options: { participantPane: true, topic: false },
+        // To prevent stale composite issues we are using a unique value (e.g the date) as a key for this component
```

Comment:
What are the stale issues we are seeing? Looks like this `new Date()` is only executed once so wondering how this key causes the composite the composite to refresh, and how often this will cause the composite to re-render?

Reply:
The issue was if you don't provide a key and you are doing some interesting app navigation to go from one screen to another, React will try to use the cached component. Now that we are introducing a key that by definition has a unique key then it will force a new React component to be rendered. This has implications when we want to make sure we have fresh chat messages and not stale ones. If we were using the older react component it would be thinking it is up to date and for whatever reason it is not getting new signals. By using the newer react component it will know there is more to get and will always be up to date in terms of chat messages. 

This was a stop gap that Nan came up with and perhaps we should be reviewing this as a larger feature in the future.

Reply:
Did you mean https://gist.github.com/prprabhu-ms/39d64aa5cbb8ed8054de9b4a086aa2ec

Reply:
Yes how React chooses to reconcile components. That was a really awesome gist.